app/assets/javascripts/pages/templates/blank.html.slim

```diff
@@ -0,0 +1,3 @@
+// Just a big white block that takes up the whole screen.
+// Should only be used for redirects.
+div.topfocus style="position:fixed;padding:0;margin:0;top:0;left:0;z-index:104;width: 100%;height: 100%;background:rgba(255,255,255);"
```

Comment:
We have the same blank loading screen on page transitions on prod too. Is there a way to reuse the same template we use from that? I have no idea how that's actually setup.

Reply:
I don't think that's actually the same thing. In prod the loader between pages is actually translucent I think, only the first load is blank, and that's because the template hasn't loaded behind it yet.

If there is an actual blank template I've been unable to find it